RecoLocalCalo/HcalRecProducers/src/HBHEPhase1Reconstructor.cc

```diff
@@ -365,6 +365,11 @@ void HBHEPhase1Reconstructor::processData(const Collection& coll,
 
 	int pulseShapeID=paramTS_->getValues(cell.rawId())->pulseShapeID();
 
+	// needed for the dark current in the M2
+	double darkCurrent = cond.getHcalSiPMParameter(cell)->getDarkCurrent();
+	double FCByPE = cond.getHcalSiPMParameter(cell)->getFCByPE();
```

Comment:
start local variables and data members with lower case letter. Capitalization is reserved for class names.